src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml

```diff
@@ -98,16 +98,23 @@
                 <a id="report-summary-continue" class="button" asp-controller="Report" asp-action="Confirm">Continue</a>
             }
             else if (Model.IsValidForSubmission && !Model.UserCanSubmitReports && !Model.IsReadOnly)
-            { 
+            {
                 <a id="report-summary-continue" class="button" asp-controller="Report" asp-action="EditComplete">Continue</a>
             }
             else
             {
-                    if (Model.Report.Submitted)
+                if (Model.Report.Submitted)
+                {
+                    if (Model.UserCanEditReports && Model.Report.Period.Equals(Model.CurrentPeriod))
+                    {
```

Comment:
I don't like the amount of logic in the UI...not entirely sure how I'd do it better, maybe move the logic up to the model so it can be tested?

Reply:
Agreed. there has been some effort in other views to push logic back (the actions and messages for the homepage spring to mind).

I'll admit this one was just a least change to get the feature in, sorry :(

The intention is to do quite a bit of re-factoring before tackling the multi-year features.